test/acceptance/framework/flags/flags.go

```diff
@@ -48,6 +50,7 @@ func (t *TestFlags) init() {
 		"If true, the tests will not clean up resources they create when they finish running."+
 			"Note this flag must be run with -failfast flag, otherwise subsequent tests will fail.")
 	flag.StringVar(&t.flagECSClusterARN, flagECSClusterARN, "", "ECS Cluster ARN.")
+	flag.StringVar(&t.flagLaunchType, flagLaunchType, "", "The ECS launch type to test: Fargate or EC2.")
```

Comment:
```suggestion
	flag.StringVar(&t.flagLaunchType, flagLaunchType, "", "The ECS launch type to test: 'FARGATE' or 'EC2'.")
```

Also should this default to fargate?